master/internal/rm/kubernetesrm/pod.go

```diff
@@ -292,22 +213,22 @@ func (p *pod) receivePodStatusUpdate(ctx *actor.Context, msg podStatusUpdate) er
 	case cproto.Starting:
 		// Kubernetes does not have an explicit state for pulling container images.
 		// We insert it here because our  current implementation of the trial actor requires it.
```

Comment:
do we still have a trial actor? (I think Erik's working on removing it, so makes sense to update this comment -- or leave it depending on what order the refactors are merged)